docker/salt/tethyscore.sls

```diff
@@ -101,6 +93,20 @@ Generate_Tethys_Settings_TethysCore:
         {{ OTHER_SETTINGS }}
     - unless: /bin/bash -c "[ -f "{{ TETHYS_PERSIST }}/setup_complete" ];"
 
+{% if TETHYS_DB_ENGINE == 'django.db.backends.postgresql' %}
```

Comment:
I don't think these should be conditional. Even if using SQLite a user may want to set the `TETHYS_DB_NAME`. I needed to add the following to support SQL Server. 

```jinja
        {%- if TETHYS_DB_OPTIONS %}
        --set DATABASES.default.OPTIONS {{ TETHYS_DB_OPTIONS }}
        {%- endif %}
```

I guess I'm fine keeping this as is for now and saying that we only support SQLite and PostgreSQL. I can make modifications later to support other Databases, web-servers, and base images.

Reply:
Added conditional for all the different db settings when setting up all the portal config settings:

https://github.com/tethysplatform/tethys/commit/fcd5a9140214a0d0978c6a1a2a7aee8eee1773f6